server/services/licenceWithConditionsBuilder.ts

```diff
@@ -1,161 +1,166 @@
+  populateLicenceWithApprovedConditions(licence: Licence) {
+    return this.populateLicenceWithConditions(licence, null, true)
   }
 
-  const licenceAdditionalConditions = licence?.licenceConditions?.additional
-  const bespokeConditions = licence?.licenceConditions?.bespoke || []
-  const conditionsOnLicence = !isEmpty(licenceAdditionalConditions) || bespokeConditions.length > 0
-
-  if (!conditionsOnLicence) {
-    return licence
-  }
+  populateLicenceWithConditions(licence: Licence, errors = {}, approvedOnly = false) {
+    // could be undefined, 'No' or 'Yes'
+    if (licence?.licenceConditions?.standard?.additionalConditionsRequired !== 'Yes') {
+      return licence
+    }
 
-  const conditionIdsSelected = Object.keys(licenceAdditionalConditions)
-  const selectedConditionsConfig = getAdditionalConditionsConfig(CURRENT_CONDITION_VERSION).filter((condition) =>
-    conditionIdsSelected.includes(condition.id)
-  )
+    const licenceAdditionalConditions = licence?.licenceConditions?.additional
+    const bespokeConditions = licence?.licenceConditions?.bespoke || []
+    const conditionsOnLicence = !isEmpty(licenceAdditionalConditions) || bespokeConditions.length > 0
 
-  const abuseAndBehavioursConditions = licence?.licenceConditions?.additional?.COMPLYREQUIREMENTS?.abuseAndBehaviours
+    if (!conditionsOnLicence) {
+      return licence
+    }
 
-  if (Array.isArray(abuseAndBehavioursConditions)) {
-    Object.assign(
-      abuseAndBehavioursConditions,
-      abuseAndBehavioursConditions.map((condition, index) => (index > 0 ? ` ${condition}` : condition))
+    const conditionIdsSelected = Object.keys(licenceAdditionalConditions)
+    const selectedConditionsConfig = getAdditionalConditionsConfig(this.conditionVersion).filter((condition) =>
+      conditionIdsSelected.includes(condition.id)
     )
-  }
 
-  return populateAdditionalConditionsAsObject(licence, selectedConditionsConfig, errors, approvedOnly)
-}
+    const abuseAndBehavioursConditions = licence?.licenceConditions?.additional?.COMPLYREQUIREMENTS?.abuseAndBehaviours
+
+    if (Array.isArray(abuseAndBehavioursConditions)) {
+      Object.assign(
+        abuseAndBehavioursConditions,
+        abuseAndBehavioursConditions.map((condition, index) => (index > 0 ? ` ${condition}` : condition))
+      )
+    }
 
-// Only exported for testing
-export function populateAdditionalConditionsAsObject(
-  rawLicence: Licence,
-  selectedConditionsConfig,
-  inputErrors = {},
-  approvedOnly = false
-) {
-  const { additional, bespoke } = rawLicence.licenceConditions
-
-  const additionalConditionsJustification =
-    rawLicence?.licenceConditions?.conditionsSummary?.additionalConditionsJustification
-  const getObjectForAdditional = createAdditionalMethod(rawLicence, selectedConditionsConfig, inputErrors)
-
-  const populatedAdditional = Object.keys(additional)
-    .sort(orderForView(getAdditionalConditionsConfig(CURRENT_CONDITION_VERSION).map((condition) => condition.id)))
```

Comment:
The main gist of this PR is to remove it's direct reliance on `CURRENT_CONDITION_VERSION` as this will be read from the licence instead